### DISCUSSION TITLE
Multiple MX record from different Email server provider

### DISCUSSION THREAD

**Discussion by shanmugad** [modoboa/modoboa#2732] [General]
Hi

Can i have a multiple MX record from different mail providers?

Example,

**Zoho Mail**

```
user1@example.com 
user2@example.com
```

**Google Webmail**

```
user3@example.com
user4@example.com
```

**Modoboa Custom server**

```

user5@example.com
user6@example.com 
```

Is this possible?

**Comment by Dr-Ari-Gami** [comment]
To my knowledge of how DNS works, no, I do not believe so. 

You can, however, set subdomains to each provider. For instance:
```
user1@zoho.example.com
user2@zoho.example.com
user3@gmail.example.com
user4@gmail.example.com
user5@modoboa.example.com
user6@modoboa.example.com
